<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personal_document</class>
 <widget class="QDialog" name="personal_document">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>7</y>
     <width>295</width>
     <height>150</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>295</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="qq_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>我总肚子饿</string>
     </property>
    </widget>
    <widget class="QPushButton" name="level_1">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_2">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_3">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_4">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_6">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_8">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_7">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_5">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_9">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_10">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_11">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_12">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_13">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level_14">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="turn_2_left">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>115</y>
      <width>10</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="turn_2_right">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>115</y>
      <width>10</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="icons_index">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>231</width>
      <height>50</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="qqaixin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="ledoudaren">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="qqzone">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="photos">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="app_daren">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="lessons">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="farm">
      <property name="geometry">
       <rect>
        <x>143</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="camera">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="lol">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="yedianzhiwang">
      <property name="geometry">
       <rect>
        <x>187</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
